sharing/sharingPane.js

```diff
@@ -13,7 +13,7 @@ const UI = require('solid-ui')
 
 module.exports = {
 
-  icon: UI.icons.iconBase + 'noun_locked_2160665_000000.svg', // noun_locked_2160665_000000.svg padlock
+  icon: UI.icons.iconBase + 'padlock-timbl.svg', // noun_locked_2160665_000000.svg padlock
```

Comment:
Where can I find this icon? As I understand it they're usually in `solid-ui`, but I can't find this one there?

Reply:
It was part of [this PR, which was merged today](https://github.com/solid/solid-ui/pull/71). Updating to the latest version of solid-ui should fix this problem.